cmd/geth/chaincmd.go

```diff
@@ -448,9 +450,13 @@ func copyDb(ctx *cli.Context) error {
 	if len(ctx.Args()) < 1 {
 		utils.Fatalf("Source chaindata directory path argument missing")
 	}
-	if len(ctx.Args()) < 2 {
-		utils.Fatalf("Source ancient chain directory path argument missing")
+
+	// Get user-defined ancient directory if available, otherwise use default.
+	freezer := ""
```

Comment:
It's wrong. The default freezer path is `freezer = filepath.Join(root, "ancient")`. Check https://github.com/ethereum/go-ethereum/blob/master/node/node.go#L578 for more details

Reply:
I think this is resolved.